src/sinks/greptimedb/logs/config.rs

```diff
@@ -0,0 +1,183 @@
+use crate::{
+    http::{Auth, HttpClient},
+    sinks::{
+        greptimedb::{
+            default_dbname_template,
+            logs::{
+                http_request_builder::{
+                    http_healthcheck, GreptimeDBHttpRetryLogic, GreptimeDBLogsHttpRequestBuilder,
+                    PartitionKey,
+                },
+                sink::{GreptimeDBLogsHttpSink, LogsSinkSetting},
+            },
+            GreptimeDBDefaultBatchSettings,
+        },
+        prelude::*,
+        util::http::HttpService,
+    },
+};
+use std::collections::HashMap;
+use vector_lib::{
+    codecs::{encoding::Framer, JsonSerializerConfig, NewlineDelimitedEncoderConfig},
+    configurable::configurable_component,
+    sensitive_string::SensitiveString,
+};
+
+fn extra_params_examples() -> HashMap<String, String> {
+    HashMap::<_, _>::from_iter([("source".to_owned(), "vector".to_owned())])
+}
+
+/// Configuration for the `greptimedb_logs` sink.
+#[configurable_component(sink("greptimedb_logs", "Ingest logs data into GreptimeDB."))]
+#[derive(Clone, Debug, Default, Derivative)]
+#[serde(deny_unknown_fields)]
+pub struct GreptimeDBLogsConfig {
+    /// The endpoint of the GreptimeDB server.
+    #[serde(alias = "host")]
+    #[configurable(metadata(docs::examples = "http://localhost:4000"))]
+    pub endpoint: String,
+
+    /// The table that data is inserted into.
+    #[configurable(metadata(docs::examples = "mytable"))]
+    pub table: Template,
+
+    /// The GreptimeDB [database][database] name to connect.
```

Comment:
```suggestion
    /// The [GreptimeDB database][database] name to connect.
```

Can we change it to this please. GreptimeDB should be a link, but having two links next to each other would be confusing so I think it makes sense to make both words a single link.